src/components/app/MenuButtons/MetaInfo.js

```diff
@@ -440,10 +512,13 @@ function _formatDate(timestamp: number): string {
   return timestampDate;
 }
 
-export const MetaInfoPanel = explicitConnect<{||}, StateProps, DispatchProps>({
+export const MetaInfoPanel = explicitConnect<{||}, OwnProps, DispatchProps>({
   mapStateToProps: (state) => ({
     profile: getProfile(state),
     symbolicationStatus: getSymbolicationStatus(state),
+    hasProfileExtraInfo: hasProfileExtraInfo(state),
+    profileExtraInfo: getProfileExtraInfo(state),
+    markerSchemaByName: getMarkerSchemaByName(state),
```

Comment:
You don't use the property `markerSchemaByName` so you can remove it everywhere in this file.